taxonomies/categories.txt

```diff
@@ -19971,6 +20093,7 @@ de:Fertiggerichte in Dosen, Dosengerichte
 es:Comidas preparadas en conserva, Platos preparados en conserva
 fi:purkitetut ateriat
 fr:Plats préparés en conserve, plats cuisinés en conserve, plat préparé en conserve
+hu:Konzerv ételek
```

Comment:
`ERROR - hu:konzerv-ételek already is a synonym of konzervek (en:canned-meals) - cannot make a synonym of konzerv-ételek for en:canned-meals`

Reply:
Thanks, I'll remove it from "canned foods" and keep it for "canned meals"